app/components/form/SideModalForm.tsx

```diff
@@ -69,12 +80,15 @@ export function SideModalForm<TFieldValues extends FieldValues>({
       form.setError('name', { message: 'Name already exists' })
     }
   }, [submitError, form])
+  const defaultTitle = title || `${formType === 'edit' ? 'Edit' : 'Create'} ${resourceName}`
+  const label =
+    submitLabel || title || `${formType === 'edit' ? 'Update' : 'Create'} ${resourceName}`
```

Comment:
this feels a little precious to me; I imagine there's a cleaner way to do this

Reply:
Actually, looking at where we use it, this is what we had before (`submitLabel || title`), plus the string literal (`${formType === 'edit' ? 'Update' : 'Create'} ${resourceName}`), so it isn't _too_ precious.